src/main/scala/com/fulcrumgenomics/illumina/SampleSheet.scala

```diff
@@ -100,13 +100,16 @@ object SampleSheet {
 
     // get the rows (values), so skip "[Data]" and the header
     val lineNumber = preData.size + 2 // 0-based
-    val sampleDatums = postData.drop(2).zipWithIndex.map { case (line, rowNumber) =>
+    val sampleDatums = postData.drop(2).zipWithIndex.flatMap { case (line, rowNumber) =>
         val values = line.split(SplitRegex, -1)
+        // If this is an empty line, skip it
+        if (values.forall(_.trim.isEmpty)) None
```

Comment:
this skips any empty lines, even in the middle of the data section, which isn't what you wanted (lines at the end only).